services/Storage/storageAccounts/alerts.yaml

```diff
@@ -434,18 +437,14 @@
       - anonymous
     threshold: 10.0
   references:
-  - name: Identify storage accounts with no or low use
-    url: https://learn.microsoft.com/azure/storage/blobs/blob-storage-monitoring-scenarios#identify-storage-accounts-with-no-or-low-use
-  - name: Monitor the use of a container
-    url: https://learn.microsoft.com/azure/storage/blobs/blob-storage-monitoring-scenarios#monitor-the-use-of-a-container
   - name: Storage Transaction Metrics
-    url: https://learn.microsoft.com/azure/storage/blobs/monitor-blob-storage-reference#transaction-metrics
+    url: https://learn.microsoft.com/en-us/azure/storage/files/storage-files-monitoring-reference#supported-metrics-for-microsoftstoragestorageaccounts
```

Comment:
Shall we align it to the link PG requested: https://learn.microsoft.com/en-us/azure/storage/files/storage-files-monitoring-reference#supported-metrics-for-microsoftstoragestorageaccounts